speech/snippets/transcribe_batch_gcs_input_inline_output_v2.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+
+# [START speech_transcribe_batch_gcs_input_inline_output_v2]
+from google.cloud.speech_v2 import SpeechClient
+from google.cloud.speech_v2.types import cloud_speech
+
+
+def transcribe_batch_gcs_input_inline_output_v2(
+    project_id: str,
+    gcs_uri: str,
+) -> cloud_speech.BatchRecognizeResults:
+    """Transcribes audio from a Google Cloud Storage URI.
+
+    Args:
+        project_id: The GCP project ID.
+        gcs_uri: The Google Cloud Storage URI.
+
+    Returns:
+        The RecognizeResponse.
+    """
+    # Instantiates a client
+    client = SpeechClient()
+
+    config = cloud_speech.RecognitionConfig(
+        auto_decoding_config=cloud_speech.AutoDetectDecodingConfig(),
+        language_codes=["en-US"],
+        model="long",
+    )
+
+    file_metadata = cloud_speech.BatchRecognizeFileMetadata(uri=gcs_uri)
+
+    request = cloud_speech.BatchRecognizeRequest(
+        recognizer=f"projects/{project_id}/locations/global/recognizers/_",
+        config=config,
+        files=[file_metadata],
+        recognition_output_config=cloud_speech.RecognitionOutputConfig(
+            inline_response_config=cloud_speech.InlineOutputConfig(),
+        ),
+    )
+
+    # Transcribes the audio into text
+    operation = client.batch_recognize(request=request)
+
+    print("Waiting for operation to complete...")
+    response = operation.result(timeout=120)
```

Comment:
Is this a required timeout? Consider adding in an explanation that might help in case users ask why this is hardcoded to 120 seconds.

Reply:
I was going off of other samples, I'm not sure what the default time out is and I'm not sure if this time out is needed. I'll evaluate and update in the next PR.